packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx

```diff
@@ -93,7 +85,7 @@ function Branding({ application, isActive }: Props) {
       return;
     }
 
-    reset(formatResponseDataToForm(data));
+    reset(data);
```

Comment:
I think the format is still needed before resetting the data

Reply:
fixed in https://github.com/logto-io/logto/pull/6209/files#diff-15411e9b879387061e0776cdacf152f5bc1ff7ccddd689cd0554d7b86b309313